assets/parca-agent.yaml

```diff
@@ -0,0 +1,240 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    pod-security.kubernetes.io/audit: privileged
```

Comment:
why do we need privileged annotation?

Reply:
parca agent injects `eBPF` programs to every resource on the cluster to do profiling. In order for those programs to access the low level(i.e kernel) syscalls info, priviledged access is required.